src/config/index.ts

```diff
@@ -0,0 +1,22 @@
+import camelCase from 'lodash/camelCase';
+
+interface ConfigProps {
+  [key: string]: string;
+}
+
+const envConfig = process.env;
+const configurationPrefix = 'REACT_APP_';
+
+const convertConfig = () => {
+  const config: ConfigProps = {};
+
+  Object.keys(envConfig).forEach((key: string) => {
+    const newKey = camelCase(key.replace(configurationPrefix, '').toLowerCase());
+
+    config[newKey] = envConfig[key] || '';
+  });
+
+  return config;
+};
+
+export const Config = convertConfig();
```

Comment:
can we create a function `config` and return it directly instead of calling via convertConfig() function and execute it

Reply:
@nvminhtue thanks, updated in [800f4ab](https://github.com/manh-t/react-survey/pull/46/commits/800f4aba172dd271928aa7b3f5e24bd0a60f05f4)